misc/userscripts/open_download

```diff
@@ -94,7 +95,7 @@ fi
 line=$(printf "%s\n" "${entries[@]}"  \
        | crop-first-column 55 \
        | column -s $'\t' -t \
-       | $ROFI_CMD "${rofi_default_args[@]}" $ROFI_ARGS) || true
+       | $ROFI_CMD "${rofi_default_args[@]}" "$ROFI_ARGS") || true
```

Comment:
`$ROFI_ARGS` should probably not be quoted so it can expand if there are multiple arguments.